src/psyclone/transformations.py

```diff
@@ -1124,8 +1135,10 @@ def apply(self, node):
         # can be run in parallel
         colour_loop = node.__class__(parent=colours_loop, loop_type="colour")
         colour_loop.field_space = node.field_space
+        colour_loop._field_name = node._field_name
```

Comment:
This is a protected access. Create an accessor function?

Reply:
There already was one - I've fixed this to actually use it!

Reply:
Great.